neuralmonkey/decoding_function.py

```diff
@@ -1,59 +1,93 @@
 import tensorflow as tf
 
-def attention_decoder(decoder_inputs, initial_state, attention_objects,
-                      embedding_size, cell, output_size=None,
-                      loop_function=None, dtype=tf.float32, scope=None):
+from neuralmonkey.logging import debug
 
-    if output_size is None:
-        output_size = cell.output_size
+def attention_decoder(decoder_inputs, initial_state, attention_objects,
```

Comment:
Klidně. Nechal bych ale v názvu attention. Taky bych nějak zkusil oddělit attentive decoder a normální. Ale asi někde jinde než tady.. Kvůli tý hidden vrstvě, kterou si to asi nechceš komplikovat, když tam ten attention nemáš. Vůbec je to taková featura navíc mi přijde. Co udělat novej dekodér s attentionama, kterej by podědil od toho obyčejnýho?